tests/integration/test_base_git.py

```diff
@@ -66,7 +66,7 @@ def test_get_output_from_action_defined_in_sandcastle():
     packit_repository_base = PackitRepositoryBase(
         config=c, package_config=PackageConfig(actions={ActionName.pre_sync: echo_cmd})
     )
-    packit_repository_base.local_project = LocalProject()
+    packit_repository_base.local_project = LocalProject(working_dir="/tmp")
```

Comment:
What was the problem here?

Reply:
This is in `test_get_output_from_action_defined_in_sandcastle()` and the problem was that `PackitRepositoryBase.local_project.working_dir` was `None` and [sandcastle/MappedDir.init() then failed on `self.local_dir: Path = Path(local_dir)`](https://github.com/packit/sandcastle/blob/master/sandcastle/api.py#L98) because the `local_dir` was `None` (passed from the `PackitRepositoryBase.local_project.working_dir`).

Previously it was probably `""` and thus did not fail.

Reply:
Aha, ok. Thanks for the explanation.